docs/api/model_selection/validation_curve.rst

```diff
@@ -27,7 +27,7 @@ In our first example, we'll explore using the ``ValidationCurve`` visualizer wit
     y = data[targets[0]]
 
     viz = ValidationCurve(
-        DecisionTreeRegressor(), ax=ax, param_name="max_depth",
```

Comment:
Deleted `ax=ax` since it throws an error because `ax` is not defined.

Reply:
@Kautumn06 If you wanted to, you could define ax, similarly to [Line 31 - 34](https://github.com/DistrictDataLabs/yellowbrick/blob/c5355eec04f51f796f8658d8b04f2e6a043ebb4c/docs/api/model_selection/validation_curve.py#L32-L44) of the docs/api/model_selection/validation_curve.py